CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ _**Version 1.5.1:**_
 
 - Fix to readthedocs building after system packages no longer imported [#336](https://github.com/tobac-project/tobac/pull/336)
 - The `strict_thresholding` option in feature detection now works correctly for detecting minima, and produces the same results as without strict thresholding if the `n_min_threshold` is a scalar value [#316](https://github.com/tobac-project/tobac/pull/316)
+- Fix to Jupyter Notebook CI that was timing out due to installing packages with `conda`, switched to `mamba` to resolve. [#340](https://github.com/tobac-project/tobac/pull/340)
```

Comment:
Previous CI changes have been under repository enhancements rather than bug fixes
```suggestion

**Repository Enhancements**
- Fix to Jupyter Notebook CI that was timing out due to installing packages with `conda`, switched to `mamba` to resolve. [#340](https://github.com/tobac-project/tobac/pull/340)
```

Reply:
Good catch; accepted your suggestion.